landoapi/projects.py

```diff
@@ -85,38 +96,26 @@ def get_project_phid(
         none_when_empty=allow_empty_result,
     )
 
-    if project is None:
-        logger.warning(f"Could not find a project phid", extra=dict(slug=project_slug))
-        return None
-
-    return phabricator.expect(project, "phid")
+    value = phabricator.expect(project, "phid") if project else None
+    cache.set(key, value, timeout=DEFAULT_CACHE_KEY_TIMEOUT_SECONDS)
+    return value
 
 
-@cache.cached(key_prefix=SEC_PROJ_CACHE_KEY, timeout=DEFAULT_CACHE_KEY_TIMEOUT_SECONDS)
```

Comment:
All the now-unused cache key constants can be removed, `SEC_PROJ_CACHE_KEY`, `CHECKIN_PROJ_CACHE_KEY` etc.